packages/framework/aqueduct/src/demo.ts

```diff
@@ -0,0 +1,163 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IFluidDataStoreRuntime } from "@fluidframework/datastore-definitions/internal";
+import {
+	SharedDirectory,
+	SharedMap,
+	type ISharedDirectory,
+} from "@fluidframework/map/internal";
+import type { ISharedObject } from "@fluidframework/shared-object-base/internal";
+import { SharedTree, type ITree } from "@fluidframework/tree/internal";
+
+import type { IDelayLoadChannelFactory } from "./channel-factories/index.js";
+import {
+	MigrationDataObjectFactory,
+	type MigrationDataObjectFactoryProps,
+} from "./data-object-factories/index.js";
+import {
+	dataObjectRootDirectoryId,
+	MigrationDataObject,
+	treeChannelId,
+	type ModelDescriptor,
+} from "./data-objects/index.js";
+
+//* NOTE: For illustration purposes.  This will need to be properly created in the app
+declare const treeDelayLoadFactory: IDelayLoadChannelFactory<ITree & ISharedObject>;
+
+// Model shapes
+interface TreeModel {
```

Comment:
Have we considered applying this to existing example apps in FF? I think for example `examples/data-objects/diceroller` is a good one that uses `DataObject` and `examples/data-objects/table-tree` is a good one that uses `TreeDataObject`.